src/App.jsx

```diff
@@ -14,57 +15,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [currentGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {!currentGood ? 'No goods selected' : `${currentGood} is selected`}
+        {currentGood && (
+          <button
+            onClick={() => {
+              setGood('');
+            }}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              className={
+                currentGood === good ? 'has-background-success-light' : ''
+              }
+              data-cy="Good"
+              key={good}
             >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+              {currentGood !== good ? (
+                <td>
+                  <button
+                    onClick={() => {
+                      setGood(good);
+                    }}
+                    data-cy="AddButton"
+                    type="button"
+                    className="button"
+                  >
+                    +
+                  </button>
+                </td>
+              ) : (
+                <td>
+                  <button
+                    onClick={() => {
+                      setGood('');
+                    }}
+                    data-cy="RemoveButton"
+                    type="button"
+                    className="button is-info"
+                  >
+                    -
+                  </button>
+                </td>
+              )}
```

Comment:
You don't need to copypaste `td` component

Reply:
It is also better not to copypaste `button` comment, and just to alter it's properties as needed

```suggestion
 <td>
  <button
     onClick={() => {
        setGood(currentGood !== good ? good : '');
     }}
     type="button"
     data-cy={currentGood !== good ? "AddButton" : "RemoveButton"}
     className={...}
  >
      {...} 
   </button>
</td>